hp/elitebook/2560p/igpu.nix

```diff
@@ -0,0 +1,28 @@
+{ config, pkgs, lib, ... }:
+with lib;
+let
+  xcfg = config.services.xserver;
+  cfg = config.hardware.hp.elitebook.graphics;
+in
+{
```

Comment:
Also I would be interested why you want to toggle the iGPU? Does the board also comes with dedicated graphic cards?

Reply:
Actualy, I don't want to toggle the iGPU, but only the hardware acceleration (vaapi, etc).
Someone might want to disable it for some reason.

Reply:
Oh wait, I'm stupid.